.github/workflows/pr-update.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: maxkomarychev/pr-updater-action@v1.0.1
+      - uses: castastrophe/actions-pr-auto-update@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          include_drafts: false
```

Comment:
Configurable feels much better!

We should document this option somewhere (and also communicate it to the team)

Reply:
Great point. The documentation is written up here but I'll add a comment to that effect in here.